pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -60,6 +60,10 @@ python3Packages.buildPythonApplication rec {
 
     # fixed by https://github.com/Backblaze/B2_Command_Line_Tool/pull/915
     "TestRmConsoleTool"
+
+    "TestConsoleTool"
+    "TestNonUTF8TerminalSupport"
+    "TestVersionConsoleTool"
```

Comment:
These failures are probably due to `pytest 7 -> 8`.
Could you please try replacing `pytestCheckHook` with `(pytestCheckHook.override { pytest = pytest_7; })`?
Then, in general, when you disable a test, you need to write a reason.

Reply:
This worked! I pushed a new commit to the PR with these changes (and removing the unnecessarily disabled tests). Let me know if I need to make a new PR or change anything else.